backend/proto/bayesian/rating.py

```diff
@@ -0,0 +1,92 @@
+import glicko2
+import math
+
+
+class Glicko:
+    """ Struct like structure representing a glicko2 rating.
+    """
+
+    def __init__(self, rating=1500.0, rd=350.0, vol=0.06):
+        """Parameters
+        ----------
+        rating : float
+            expected value of the rating
+            
+        rd : float
+            rating deviation
+        
+        vol : float
+            volatility
+        """
+        self.rating = rating
+        self.rd = rd
+        self.vol = vol
+
```

Comment:
Le string formatting avec % est deprecated. Tu devrais utiliser "{}".format 
voir http://stackoverflow.com/questions/14753844/python-3-using-s-and-format